src/common/config/state/info.js

```diff
@@ -45,5 +45,6 @@ export function getInfo (id) {
 export function setInfo (id, obj) {
   if (!id) throw new Error('All info objects require an agent identifier!')
   _cache[id] = getModeledObject(obj, model)
-  gosNREUMInitializedAgents(id, _cache[id], 'info')
+  const agentInst = gosNREUMInitializedAgents(id)
+  if (agentInst) agentInst.info = _cache[id]
```

Comment:
Assuming it exists on the global is risky & could break; this ties back to https://github.com/newrelic/newrelic-browser-agent/pull/785#discussion_r1370360058 because it wouldn't exist yet on agent startup if not for moving the assignment up to the top. So it's not a known issue now, but some other thing could remove the reference, so it's better to just have it guarded. This'll go away when the _cache is replaced with just modifying `this.info` directly.

Reply:
I guess the designation I am calling out is that we shouldn't call it "get or set" if it doesn't actually set if not getting.

Reply:
I suppose (though that's outlined in the jsdoc). Still, I don't think this is worth the effort to split & update, keeping in mind this'll go away anyways some time later.